server/aws-lsp-codewhisperer/src/language-server/agenticChat/errors.ts

```diff
@@ -16,3 +21,15 @@ export function wrapErrorWithCode(error: unknown, code: AgenticChatErrorCode) {
     }
     return new AgenticChatError(String(error), code)
 }
+
+export function isInputTooLongError(error: unknown): boolean {
+    if (error instanceof AgenticChatError && error.code === 'InputTooLong') {
+        return true
+    }
+
+    if (error instanceof Error) {
```

Comment:
This seems pretty fragile (breaks if the backend changes their error message wording), but I don't really see another way.

Reply:
+1. Let me add an explicit comment in the code here